### PR TITLE
Php exp

### PR DESCRIPTION
**Pull request by colleend**
Should fix issue 
https://github.com/returntocorp/semgrep/issues/1808.

### CODE REVIEW

File: lang_php/parsing/ast_php.mli.nw [hunk 1]
```diff
@@ -567,7 +567,7 @@ and logic expressions inherited from C:
  and binaryOp    = Arith of arithOp | Logical of logicalOp 
    <<php concat operator>>
        and arithOp   = 
-         | Plus | Minus | Mul | Div | Mod
+         | Plus | Minus | Mul | Div | Mod | Exp
```
Comment (aryx) [inline lang_php/parsing/ast_php.mli.nw:570]: Sorry I should have told you, but you don't need to update those .nw files. They are noweb files used for literate progrmaming but their content can be automatically synced for the code, so no need to update them manually.

**Review by aryx — APPROVED** [review]
Great!
I really need also to remove the intermediate cst_php.ml like I did for Javascript because
this is getting really tedious to modify 10 files when you just need to add an operator ...

**Comment by aryx** [comment]
I'll fix the CI. Thx @colleend

**Comment by aryx** [comment]
Argh, oops:
Warning: 69 states have shift/reduce conflicts.
Warning: 99 shift/reduce conflicts were arbitrarily resolved.
This is not good ... We used to check for regressions in shift/reduce conflicts when pfff was using Makefile (@emjin actually added this checking code), but now that we switched to dune build system the check is not there, so when you modify a grammar you need to make sure this does not introduce new shift/reduce (s/r) or even worse reduce/reduce (r/r/) grammar conflicts

File: lang_php/parsing/ast_php.mli.nw (hunk 1 above)
Comment (colleend) [reply lang_php/parsing/ast_php.mli.nw:570]: ah gotcha!! thanks for the heads up 👍

**Comment by colleend** [comment]
ohhh ok yikes, I'll be sure to check next time 👍

**Pull request merged**